examples/rcbilling-demo/src/tests/main.test.ts

```diff
@@ -14,7 +14,7 @@ test.describe("Main", () => {
     // Gets all elements that match the selector
     const packageCards = await getAllElementsByLocator(page, CARD_SELECTOR);
 
-    const EXPECTED_VALUES = [/30\.00/, /19\.99/, /15\.00/];
+    const EXPECTED_VALUES = [/30[,.]00/, /19[,.]99/, /15[,.]00/];
```

Comment:
Did this to also account for other locales where commas are used as decimal delimiters.